components/Base/FileUpload.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <div>
+    <label
+      class="flex flex-col items-center justify-center w-full bg-gray-100 border-[2px] border-dashed h-20 rounded-3xl"
+      :accept="accept"
```

Comment:
I'm not sure that accept should be here 😅 For example I can drag & drop `.torrent` file, see below

Reply:
This is generic file upload, so accepts all files. The issue is I have not bound a prop in next component `License upload` that should have limit it to document and images